guides/CONTRIBUTING.md

```diff
@@ -73,14 +71,12 @@ Submitting Pull Requests that resolve issues is great way to contribute to Scree
 > [!tip]
 > For commits and pull request names we follow a [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification.
 
-We have two types of applications: pure examples (Example, FabricExample as wrappers and apps/examples for source code) and examples, dedicated for tests from issues / pull requests (FabricTestExample, TestsExample as wrappers and apps/test-examples for source code). The first work as a showcase of the library, the latter contains specific test cases that corresponds to GitHub issues. For example, `Test1864.tsx` corresponds to issue [#1864](https://github.com/software-mansion/react-native-screens/issues/1864). Our developer flow usually consists of creating new `Test*.tsx` file with code example that we try to fix or add. For new features we try to prepare showcases in Example app.  
+We have two types of sources: pure examples (apps/App for source code) and examples, dedicated for tests from issues / pull requests (apps/TestApp for source code). The first work as a showcase of the library, the latter contains specific test cases that corresponds to GitHub issues. For example, `Test1864.tsx` corresponds to issue [#1864](https://github.com/software-mansion/react-native-screens/issues/1864). Our developer flow usually consists of creating new `Test*.tsx` file with code example that we try to fix or add. For new features we try to prepare showcases in Example app. Example and FabricExample applications are used as wrappers for those applications.
```

Comment:
I'd rewrite last 3 sentences now they're missing the point imo. We create testing example in TestApp source and showcases in example app. The wrapper depends on arch, don't think if that's clear now for someone who didn't use it before

Reply:
https://github.com/software-mansion/react-native-screens/pull/2225/commits/a415eca814b16a2cf4bde49fd087c80fb2d24973